packages/techdocs-common/api-report.md

```diff
@@ -263,21 +251,19 @@ export const transformDirLocation: (
   target: string;
 };
 
-// Warning: (ae-missing-release-tag) "UrlPreparer" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-//
-// @public (undocumented)
+// @public
+export type UrlFactory = PreparerFactory;
```

Comment:
What's the thinking on exporting this as a separate type?

Reply:
Replaced by `PreparerConfig`, https://github.com/backstage/backstage/pull/9718/commits/e257a8f0f499672b3e696df53d0604aba9ddbf0b.